resources/public/api/conf/1.0/common/headers.yaml

```diff
@@ -56,7 +56,7 @@ components:
       in: header
       type: string
       description: |
-        Unique ID for operation tracking
+        Unique ID for operation tracking.
```

Comment:
Good. I think Ben wants us to add a full stop to the end of the error messages too (See point 3). E.g. the error message in this screenshot is still missing the '.' after "invalid":  
<img width="584" alt="image" src="https://github.com/hmrc/individuals-foreign-income-api/assets/106095575/db407bec-84c9-400d-8d7a-ae9c7a26b415">